packages/browser/rollup.bundle.config.mjs

```diff
@@ -8,6 +8,10 @@ if (targets.some(target => target !== 'es5' && target !== 'es6')) {
   throw new Error('JS_VERSION must be either "es5" or "es6"');
 }
 
+const browserPluggableIntegrationFiles = ['contextlines', 'httpclient', 'reportingobserver'];
+
+const coreIntegrationFiles = ['captureconsole', 'debug', 'dedupe', 'extraerrordata', 'rewriteframes', 'sessiontiming'];
```

Comment:
super-l: wdyt about naming this `corePluggableIntegrationFiles` for consistency?